tcmu-runner.spec

```diff
@@ -123,6 +123,9 @@ userspace libraries they like.
 %config(noreplace) %{_sysconfdir}/logrotate.d/tcmu-runner
 
 %changelog
+* Mon Nov 26 2018 Prasanna Kumar Kalever <prasanna.kalever@redhat.com>
```

Comment:
I am thinking maybe these changelog additions are not very useful, because we have the git history for all changes already. Are they for us or for users I mean?

For each release we are not updating the changelog so users are not getting the full benefit of seeing what is changing from release to release. They are just seeing what changes in the rpm which is not very useful compared to getting a list of bugs and features in each list.

I would propose just stop adding these lines and maybe for each release I will add a proper changelog for major features/bugs or maybe for each release in the changelog I will add link to the GitHub release notes which has a lis of major changes?

Reply:
As long as we are caring to update the changelog at rpm packaging (i.e. at fedora, centos, rhel), I'm okay removing this.

I will submit a cleanup patch. Thanks!